Microsoft.Toolkit.Uwp.SampleApp/SamplePages/PrintHelper/PrintHelperPage.xaml.cs

```diff
@@ -82,6 +106,58 @@ public PrintHelperPage()
             await _printHelper.ShowPrintUIAsync("UWP Community Toolkit Sample App", printHelperOptions, true);
         }
 
+        private void CustomPrint_Click(object sender, Windows.UI.Xaml.RoutedEventArgs e)
```

Comment:
Can you add this (or a simplified version of this) to the code bind file and documentation?

Reply:
OK. 
With 'documentation' you mean 'MicrosoftDocs/UWPCommunityToolkitDocs'  ?

Reply:
No, just the documentation in the docs folder https://github.com/Microsoft/UWPCommunityToolkit/blob/master/docs/helpers/PrintHelper.md

and the bind file in the sample folder:
https://github.com/Microsoft/UWPCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/PrintHelper/PrintHelperCode.bind

Reply:
And ... done.